src/main/java/net/sf/jsqlparser/statement/create/table/CreateTable.java

```diff
@@ -186,18 +188,21 @@ public String toString() {
             sql += ")";
         }
         String options = PlainSelect.getStringList(tableOptionsStrings, false, false);
-        if (options != null && options.length() > 0) {
+        if (!options.isEmpty()) {
```

Comment:
Is this really NULL safe?

Reply:
Yes, it is a StringBuilder result from the `PlainSelect#getStringList(...)`

Reply:
Ok, thanks for checking.
`isEmpty` is available on Java 8? I recall it was available in Java 11 only?

Reply:
yes it is available, since 1.6

Reply:
Ok, sorry for bothering you.

Thanks for the PR, it looks very good in my opinion, well done.
But only @wumpz can merge it eventually. Please be patient.

Reply:
Any chance to have it merged this year?

Reply:
Is the project abandoned?

Reply:
The project is very active and as accepted a few PRs since your contribution.
Thus you will need to resolve conflicts first (Pull + Merge + Resolve Conflicts + Commit + Push), before your PR can be accepted.

Thank you for understanding.